src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -275,141 +233,60 @@ private void placeBlock(IBlockState localState, @NotNull Block localBlock, @NotN
     }
 
     /**
-     * Scan the schematic and save it to the disk.
+     * Scan the structure and save it to the disk.
      *
      * @param world Current world.
      * @param from  First corner.
      * @param to    Second corner.
      * @return Message to display to the player.
      */
-    public static String saveSchematic(@Nullable World world, @Nullable BlockPos from, @Nullable BlockPos to)
+    public static String saveStructure(@Nullable World world, @Nullable BlockPos from, @Nullable BlockPos to)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "saveStructure" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)